packages/neutron-sdk/src/bindings/marketmap/types.rs

```diff
@@ -0,0 +1,79 @@
+use schemars::{JsonSchema, Map};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct Params {
+    #[serde(default)]
+    pub version: u64,
```

Comment:
m, i actually do not see `vesrion` field in params type https://github.com/skip-mev/slinky/blob/main/x/marketmap/types/params.pb.go#L26 and do see `admin` field. Do i miss something?